README.md

```diff
@@ -172,6 +174,19 @@ POST /token
 </td>
 </tr>
 
+<tr>
+<td> 400 </td>
+<td>
+
+```json
+{
+  "error": {
+    "message": "invalid parameter",
+    "explanation": "A boolean value must be provided for the create_room parameter"
```

Comment:
But `create_room` is optional right? So this error would never happen?

Reply:
@timrozum if you look at the tests below you'll see that this error covers cases where `create_room` is passed `'a string'` for example.

Reply:
Oh I see thanks!